Client/Clique/Modules/EventDetail/EventDetailViewController.swift

```diff
@@ -143,32 +142,28 @@ private extension EventDetailViewController {
         }
     }
     
-    @objc func registerToEventAlert(){
-        let refreshAlert = UIAlertController(title: Constants.Alerts.confirmationTitleMessage, message: alertMessage, preferredStyle: UIAlertController.Style.alert)
-
-        refreshAlert.addAction(UIAlertAction(title: Constants.Alerts.defaultOKActionTitle, style: .default, handler: { (action: UIAlertAction!) in self.callRegisterToEventService()
-        }))
-
-        refreshAlert.addAction(UIAlertAction(title: Constants.Alerts.defaultCancelActionTitle, style: .cancel, handler: { (action: UIAlertAction!) in
-        }))
-
-        present(refreshAlert, animated: true, completion: nil)
+    @objc func registerToEventAlert() {
+        let okAction = UIAlertAction(title: Constants.Alerts.defaultOKActionTitle,
+                                     style: .default, handler: {(action: UIAlertAction!) in self.callRegisterToEventService()})
+        let cancelAction = UIAlertAction(title: Constants.Alerts.defaultCancelActionTitle,
+                                         style: .default, handler: .none)
+        showAlert(message: alertMessage, actions: [okAction, cancelAction])
```

Comment:
If you are dividing your parameters in multiple rows each param must be in it's own row!